master/security/host-forwarded-traffic.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Apply policy to forwarded traffic
+---
+
+### Big picture
+
+Enforce {{site.prodname}} policy on traffic transiting a host that is used as a router or NAT gateway.
+
+### Value
+
+If your host has multiple network interfaces, and is configured as a router, or NAT gateway between two different networks, you may want to enforce policy on traffic as it moves between the networks. In this configuration, often neither the source or destination is a {{site.prodname}} endpoint, so policy enforcement at the endpoint is not available. You can centrally manage the firewall policy on a fleet of such hosts using the same policy language as the rest of {{site.prodname}}.
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- **Host Endpoint**
+- **GlobalNetworkPolicy**
+  - applyOnForward setting
+  - preDNAT setting
+
+### Concepts
+
+#### Workload endpoints and host endpoints
+
+The following figure shows a host with two network interfaces: eth0 and eth1. We call these **host endpoints (HEPs)**. The host also runs two guest workloads (VMs or containers). We call the virtual interfaces to the guests, **workload endpoints (WEPs)**.  Each has a corresponding configuration object on the {{site.prodname}} API called HostEndpoint and WorkloadEndpoint, respectively. 
+
+The `HostEndpoint API` object is optional, and Calico does not enforce any policy on the HEP if the API object is missing. The `WorkloadEndpoint API` object is required, and is automatically managed by the cluster orchestrator plugin (for example, Kubernetes or OpenStack).
+
+Several connections are shown in the figure, numbered 1 through 4. For example, connection 1 ingresses over HEP eth0, is forwarded, and then ingresses Workload A’s WEP. Calico policies select which WEPs or HEPs they apply to. So, for example an ingress policy that selects Workload A’s WEP will apply to connections as shown in number 1.
+
+![Host-forward-traffic]({{site.baseurl}}/images/host-forward-traffic.png)
```

Comment:
![image](https://user-images.githubusercontent.com/170076/62263932-c15b4900-b3d2-11e9-94e4-cd78f99b8095.png)

Looks like a typo here - there should be an `eth0` and an `eth1`.

Reply:
Great catch! Fixed.